frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -276,10 +279,13 @@ private void processSuccessfulCodeRequest(
                     true);
 
             if (configurationService.isAuthenticationAttemptsServiceEnabled()) {
```

Comment:
Could this go in a clarifying method called, for example, ensureSignedInUserCanReauthenticateByClearingErrorCounts?